elements/pfe-card/test/pfe-card.spec.ts

```diff
@@ -195,4 +195,19 @@ describe('<pfe-card>', function() {
       });
     });
   });
+
+  describe('accessible', () => {
+    it('should be accessible in both the light and shadow dom', async () => {
+      const element = await createFixture<PfeCard>(TEMPLATES.card1);
+
+      expect(element).to.be.accessible();
+      expect(element.shadowRoot).to.be.accessible();
+    });
+
+    it('should have an article element wrapper in the shadow dom', async () => {
+      const element = await createFixture<PfeCard>(TEMPLATES.card1);
+
+      expect(element.shadowRoot?.querySelector('article')).not.to.be.undefined;
```

Comment:
This should not query the shadow root. Use the a11y snapshot instead

Reply:
So when I was looking at using a11ysnapshot there did not appear to be a way to tell whether or not the element has an article element present.  If the article element is present or it's not I see a role of 'WebArea' so I'm not sure if that would be the best test case here.  

Here's the output I'm seeing

```
{
        role: 'WebArea',
        name: '',
        children: [
          { role: 'heading', name: 'Card 1', level: 2 },
          { role: 'text', name: 'This is pfe-card.' },
          { role: 'text', name: 'Text in footer' }
        ]
      }
```

Though there's also the possibility I'm just not using the plugin correctly lol

Reply:
https://github.com/modernweb-dev/web/issues/2083
it's not you. 

Do your best until then, but try to avoid querying the shadow root